internal/types.ts

```diff
@@ -30,6 +30,7 @@ export type FoxgloveMessageField = {
   required?: true;
   description: string;
   protobufFieldNumber?: number;
+  defaultValue?: string;
```

Comment:
I think the default value for numeric fields probably shouldn't be a string, it should be the actual type of the field (`number` in the cases you've added, although I suppose we could have defaults for string fields). Then it can be up to each generator to know how to translate a number into valid syntax in its own language.

Reply:
yeah I honestly am not sure what would be the right way to type that aligned with the `type` type, without putting it inside of that, but maybe that makes more sense since we might not want defaults of nested schemas?